SUPPORT.md

```diff
@@ -0,0 +1,22 @@
+<!-- https://help.github.com/en/github/building-a-strong-community/adding-support-resources-to-your-project -->
+
+# Support
+
+Having trouble? Please consult these official resources in order of appearance.
+
+## Resources
+
+**[open.inf.is][docs-site] on the Web:** The OpenINF website. It has docs and
+tutorial resources.
+
+**[`#openinf`][matrix-channel] on matrix.org:** The OpenINF Matrix support channel
+on the _matrix.org_ network. It’s a synchronous chat channel for lightweight
+communication. Good uses of it are for asking questions, casual conversation,
+collaboration, and quick exchanges.
+
+**[@openinf][twitter-account] on Twitter:** The OpenINF support account on
+Twitter.
+
+[docs-site]: https://open.inf.is/
+[matrix-channel]: https://matrix.to/#/#openinf:matrix.org
+[twitter-account]: https://twitter.com/openinf
```

Comment:
suggest, unified brand and username as one

https://github.com/OpenINF
https://twitter.com/OpenINF

Reply:
I am trying to remember why, but when I had it like that before, it caused a big headache for the GitHub one; let me think about that and try to remember what the issue was&hellip;

Reply:
```suggestion
[twitter-account]: https://twitter.com/OpenINF
```